script/clear_external_cache.pl

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use MusicBrainz::Server::Context;
+
+# This script can be used to clear everything in memcached. It assumes that
+# the CACHE_MANAGER_OPTIONS DBDefs setting resembles the default one (so it
+# has an "external" profile pointing to Cache::Memcached::Fast, and
+# default_profile is set to "external").
+
+my $c = MusicBrainz::Server::Context->create_script_context;
+$c->cache->flush_all;
```

Comment:
Okay, there's a new script in https://github.com/mwiencek/musicbrainz-server/commit/1cc98c5c72ae31f54701802501caba66c6a5cb6f that deletes things updated in the last 2 hours. I removed the old script that clears the entire cache.